molecule/shared/prepare.yml

```diff
@@ -0,0 +1,22 @@
+- name: preconfigure users
+  hosts: all
+  tasks:
+    - name: create user derp
+      user:
+        name: derp
```

Comment:
Shouldn't this be replaced with `"{{ passwordless_ssh_user }}"` or `"{{ passwordless_ssh_become_user }}"`?

Reply:
This is setting up the text fixture data and, as such, cannot reference the role's internal parameters.